pallets/anchors/src/lib.rs

```diff
@@ -100,6 +100,9 @@ pub mod pallet {
 	{
 		/// Type representing the weight of this pallet
 		type WeightInfo: WeightInfo;
+
+		#[pallet::constant]
+		type MaxBound: Get<u32>;
```

Comment:
Maybe this name can be renamed to something more relevant for the runtime configurator in order to know what is exactly this bound for. Something like `MaxRootOfChildTrieSize`.

Because our `Vec<u8>` is storing the child root (docs say that the hashing algorithm is defined by the Block), seems like its size is something known. I mean, instead of passing a new argument, could we extract it from the current hashing algorithm used?

Reply:
@mustermeiszer I just see your comment below regarding the size, what do you think about this? Could it be possible?

Reply:
Love the idea, but is is defined not by `T::Hash`, but by the `type Hash` of `trait Header`.  As the externalities hasher defines this hash type.

Reply:
Done in https://github.com/centrifuge/centrifuge-chain/pull/980